.env.prod.example

```
@@ -0,0 +1,38 @@
+# GHCR image
+IMAGE_REPOSITORY=ghcr.io/YOUR_GITHUB_OWNER/ejib-backend
+IMAGE_TAG=IMAGE_TAG=CHANGE_ME_IMMUTABLE_TAG   # 예: v1.2.3 또는 sha-<shortsha>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)